apix-docker/231/docker-compose.yml

```diff
@@ -2,9 +2,10 @@ version: '2'
 
 services:
   alfresco-core:
-    image: ${DOCKER_IMAGE}
+    image: ${DOCKER_IMAGE:-local}
```

Comment:
I don't think `local` works. Try `docker pull local` to test. You want `private.docker.xenit.eu/...:local`

Reply:
This is to add a tag locally so that the ./gradlew :...:composeDown works. I  believe this solves the issues where the local docker-compose needs a label set on the docker image to bring the stack down. The simple :-local adds a tag here that fills-out the label. 
We don't use a docker pull but the local docker build file in the CI/Local?

Reply:
No, this does not add anything, it sets. It's bash syntax: https://www.baeldung.com/linux/bash-variable-assign-default. This means if DOCKER_IMAGE is unset, you will end up with:
```
services:
  alfresco:
    image: local
```
which does not work since there is no image with the name `local`.

Reply:
It is just meant to set the default value of DOCKER_IMAGE. Locally this will make composeDown fail since the default value=null. And in the CI this will not do anything since it sets the values for the DOCKER_IMAGE itself.
This is only meant to fix local build.